Could not compile program with NativeAOT

Hello! I have been trying to compile my Hello, world! application with NativeAOT however I get this error:

```
LINK : fatal error LNK1104: cannot open file '@(NetCoreAppNativeLibrary->'C:\Users\Lia's Mansion\.nuget\packages\runtime.win-x64.microsoft.dotnet.ilcompiler\6.0.0-rc.1.21420.1\framework\%(Identity)-Static.lib')' [C:\Users\Lias Mansion\source\repos\IL2CPPTests\IL2CPPTests\IL2CPPTests.csproj]
C:\Users\Lia's Mansion\.nuget\packages\microsoft.dotnet.ilcompiler\6.0.0-rc.1.21420.1\build\Microsoft.NETCore.Native.targets(352,5): error MSB3073: The command ""D:\VS2019\Community\VC\Tools\MSVC\14.29.30037\bin\Hostx64\x64\link.exe"  @"obj\release\net6.0\win-x64\native\link.rsp"" exited with code 1104. [C:\Users\Lias Mansion\source\repos\IL2CPPTests\IL2CPPTests\IL2CPPTests.csproj]
```

I really have no clue what is going on, but it happens when the DLL file is being made(with dotnet publish -r win-x64 -c Release)

Thanks for any help in advance. :D

--- a/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.props
+++ b/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.props
@@ -56,7 +56,10 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup>
       <DirectPInvoke Include="@(NetCoreAppNativeLibrary->'lib%(Identity)')" />
-      <NativeLibrary Include="@(NetCoreAppNativeLibrary->'$(IlcPath)/framework/lib%(Identity).a')" />
+      <NetCoreAppNativeLibrary Include="@(NetCoreAppNativeLibrary->'%(Identity)')">
+        <EscapedPath>$(IlcPath)/framework/lib%(Identity).a</EscapedPath>
+      </NetCoreAppNativeLibrary>
+      <NativeLibrary Include="@(NetCoreAppNativeLibrary->'%(EscapedPath)')" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetOS)' == 'OSX'">

--- a/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props
+++ b/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props
@@ -43,7 +43,10 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup>
       <DirectPInvoke Include="@(NetCoreAppNativeLibrary->'%(Identity)')" />
-      <NativeLibrary Include="@(NetCoreAppNativeLibrary->'$(IlcPath)\sdk\%(Identity).Aot$(LibrarySuffix)')" />
+      <NetCoreAppNativeLibrary Include="@(NetCoreAppNativeLibrary->'%(Identity)')">
+        <EscapedPath>$(IlcPath)\sdk\%(Identity).Aot$(LibrarySuffix)</EscapedPath>
+      </NetCoreAppNativeLibrary>
+      <NativeLibrary Include="@(NetCoreAppNativeLibrary->'%(EscapedPath)')" />
     </ItemGroup>
 
     <ItemGroup>

